.drone.yml

```diff
@@ -0,0 +1,83 @@
+---
+kind: pipeline
+type: kubernetes
+name: pr
+
+steps:
+  - name: fetch tags
+    image: docker:git
+    commands:
+      - git fetch --tags
+  - name: wait for docker
+    image: docker
+    commands:
+      - timeout 15s /bin/sh -c 'while [ ! -S /var/run/docker.sock ]; do sleep 1; done'
```

Comment:
nit: will `systemctl start docker` (w/o --no-block) not achieve the same in a simpler way?

Reply:
Not exactly, for a couple reasons:

1) Later in the drone config I start `docker:dind` as a drone "service"--a container that runs in the background throughout the entire pipeline.  The issue this `wait for docker` step solves is: other steps were starting before `/var/run/docker.sock` became available ([seen here](https://drone.gravitational.io/gravitational/robotest/9/1/3)).  Drone's [official documentation](https://docs.drone.io/pipeline/kubernetes/examples/service/docker_dind/) suggests a `sleep 5`
2) We want docker to persist across steps, because we want to build `robotest:buildbox` once and then use it for all of `lint`, `test, `build` and `containers`.
3) Lastly, systemctl isn't available in the image this is running in `docker:latest`.  I could pick a container that has systemctl, but that seems like extra weight for what this one liner accomplishes.